server/sonar-server/src/main/java/org/sonar/server/es/RecoveryIndexer.java

```diff
@@ -120,16 +117,18 @@ void recover() {
       while (!items.isEmpty()) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Reduce the total number of break and continue statements in this loop to use at most one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS135)